apps/condo/domains/user/schema/CheckUserExistenceService.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Generated by `createservice user.CheckUserExistenceService --type queries`
+ */
+
+const get = require('lodash/get')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { checkDvAndSender } = require('@open-condo/keystone/plugins/dvAndSender')
+const { GQLCustomSchema, find } = require('@open-condo/keystone/schema')
+
+const { NOT_FOUND, WRONG_FORMAT, DV_VERSION_MISMATCH } = require('@condo/domains/common/constants/errors')
+const access = require('@condo/domains/user/access/CheckUserExistenceService')
+const { ConfirmPhoneAction } = require('@condo/domains/user/utils/serverSchema')
+const { checkDailyRequestLimitCountersByIp, checkDailyRequestLimitCountersByPhone } = require('@condo/domains/user/utils/serverSchema/requestLimitHelpers')
+
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const ERRORS = {
+    TOKEN_NOT_FOUND: {
+        query: 'checkUserExistence',
+        variable: ['data', 'token'],
+        code: NOT_FOUND,
+        type: 'TOKEN_NOT_FOUND',
+        message: 'Token not found',
+    },
+    DV_VERSION_MISMATCH: {
+        query: 'checkUserExistence',
+        variable: ['data', 'dv'],
+        code: BAD_USER_INPUT,
+        type: DV_VERSION_MISMATCH,
+        message: 'Wrong value for data version number',
+    },
+    WRONG_SENDER_FORMAT: {
+        query: 'checkUserExistence',
+        variable: ['data', 'sender'],
+        code: BAD_USER_INPUT,
+        type: WRONG_FORMAT,
+        message: 'Invalid format of "sender" field value',
+        correctExample: '{ dv: 1, fingerprint: \'example-fingerprint-alphanumeric-value\'}',
+    },
```

Comment:
looks like common errors, mb move it to common consts?

Reply:
fixed